courses/free-courses-ja.md

```diff
@@ -1,14 +1,20 @@
 ### Index
 
 * [0 - 大規模公開オンライン講座(MOOC)](#0---mooc)
+* [HTML](#HTML)
```

Comment:
```suggestion
* [HTML](#html)
```

Reply:
I changed it to lowercase letters. Could you check please.